scripts/one-page.js

```diff
@@ -53,7 +53,16 @@ const getSrcContent = async src => {
       // Initial cleanup for inline strings and functions
       htm = htm.replace(/>\${/gm, '>\n${').replace(/}<\/\${/gm, '}\n</${');
 
-      const formatted = `${open}${pretty(htm)}${close}`
+      // Protect indentation of inline strings by temporarily wrapping in <pre>
+      htm = htm
+        .replace(/\n\${`/gm, '\n<pre>######${`')
+        .replace(/`}\n/gm, '`}######</pre>\n');
+
+      // Make the HTML pretty:
+      htm = pretty(htm).replace(/<pre>######|######<\/pre>/g, '');
```

Comment:
FWIW, I added the `######` to ensure we don't accidentally delete a real `<pre>` tag.

Reply:
Great Idea! 

It's out there, but I could see `######` potentially happening in a deck. Can we maybe switch to a much more obnoxious token like `SPECTACLE_ONE_PAGE_TEMP_PRE_TOKEN_START` and `SPECTACLE_ONE_PAGE_TEMP_PRE_TOKEN_STOP` or something?

Reply:
Absolutely, good idea.